content/summit2025/_index.html

```diff
@@ -0,0 +1,175 @@
+---
+url: /summit2025
+aliases: ['/summit']
+layout: index
+title: Transparency.dev Summit
+description: Event page for the 2025 Transparency.dev Summit
+---
+<section class="glue-page gmt-7 gmb-3">
+  <div class="glue-grid">
+    <div class="col-span-4 sm:col-span-10 sm:col-start-2 text-center">
+      <h1 class="headline2 gmb-2">Transparency.dev Summit</h1>
+    </div>
+  </div>
+</section>
+
+<div class="glue-page">
+  <div class="glue-grid illustration-mb">
+    <div class="col-span-4 sm:col-span-10 sm:col-start-2 ">
+      <img src="{{< rel "/images/summit2025/tdev_summit_banner.svg" >}}" class="w-full" />
+    </div>
+  </div>
+</div>
+
+<section class="glue-page">
+  <div class="glue-grid gmb-2">
+    <div class="col-span-4 sm:col-span-8 sm:col-start-3">
+      <p class="headline5 gmb-2">
+        Join us for Transparency.dev Summit in Gothenburg, Sweden on October 20-22, 2025.
+      </p>
+      <p class="headline5 gmb-2">
+        This event aims to bring together implementers, operators, and clients of real world transparency systems in order to meet peers, share best practices, and learn about the latest developments in the community.
+      </p>
+      <p class="headline5 gmb-2">
+        In addition to general transparency topics, the event will also feature dedicated sessions for the Certificate Transparency community (e.g a colocated CT day style event). 
```

Comment:
```suggestion
        In addition to a variety of transparency-log topics, the event will feature dedicated sessions for the Certificate Transparency community (e.g., a colocated CT day style event). 
```

(At least once scope the much broader "transparency" to be anchored to the core tech this conference is about.)

Reply:
(Also brought this updated sentence into the interest survey.)